Source/Tools/FEXLoader/LinuxSyscalls/Syscalls.cpp

```diff
@@ -525,6 +525,14 @@ static uint64_t Clone3Handler(FEXCore::Core::CpuStateFrame *Frame, FEX::HLE::clo
 uint64_t CloneHandler(FEXCore::Core::CpuStateFrame *Frame, FEX::HLE::clone3_args *args) {
   uint64_t flags = args->args.flags;
 
+  if (flags & CLONE_CLEAR_SIGHAND) {
+    // CLONE_CLEAR_SIGHAND was added in kernel 5.5. FEX doesn't properly support this.
+    // glibc started using this flag in 2.38 as an optimization for posix_spawn.
+    // If clone returns EINVAL or ENOSYS then it will fallback to the non-optimized path.
+    // Temporarily disable clone3 if this flag is used.
```

Comment:
> Temporarily disable clone3 if this flag is used.

What does this mean? We don't seem to temporarily disable anything here.

Reply:
That's *permanently* though as far as behavior goes.

I'm guessing what you meant with temporary is either that we hope to drop this workaround eventually OR the fact that glibc will retry after seeing the EINVAL return value.

The rest of the comment is clearly written, but the last line adds ambiguity. Just removing it seems better to me, since it's reasonably obvious that we shouldn't normally return EINVAL but do so here to make use of glibc's fallback path. Alternatively, a TODO could stress that this is just a workaround.